File: Coding/coding modul 5/PRAK504-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
# Function to reverse the digits of a number
def reverse(n):
    return int(str(n)[::-1])  # Convert the number to a string, reverse it, and convert it back to an integer

def main():
    # Read two integers A and B
    A, B = map(int, input().split())
    
    # Reverse A and B
    A = reverse(A)
    B = reverse(B)
    
    # Add the reversed numbers
    C = A + B
    
    # Reverse the sum and print it
    print(reverse(C))

# Run the main function
if __name__ == "__main__":
    main()

File: Coding/coding modul 5/PRAK504-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>

// Function to reverse the digits of a number
int reverse(int n) {
    int reversed = 0;
    while (n != 0) {
        int digit = n % 10; // Extract the last digit
        reversed = reversed * 10 + digit; // Append the digit to the reversed number
        n = n / 10; // Remove the last digit from the original number
    }
    return reversed;
}

int main() {
    int A, B;
    // Read two integers A and B
    scanf("%d %d", &A, &B);
    
    // Reverse A and B
    A = reverse(A);
    B = reverse(B);
    
    // Add the reversed numbers
    int C = A + B;
    
    // Reverse the sum and print it
    printf("%d", reverse(C));
    
    return 0;
}
